providers/os/resources/os.go

```diff
@@ -220,6 +221,16 @@ func (s *mqlOs) hostname() (string, error) {
 	return "", errors.New("cannot determine hostname")
 }
 
+func (s *mqlOs) hypervisor() (string, error) {
+	conn := s.MqlRuntime.Connection.(shared.Connection)
+	platform := conn.Asset().Platform
+
+	if res, ok := hypervisor.Hypervisor(conn, platform); ok {
+		return res, nil
+	}
+	return "", errors.New("cannot determine hypervisor")
```

Comment:
Are the cases where we don't determine anything really an error? In my case my mac is not a VM and I'd expect a null value w/o the error.

```
cnquery> os.hypervisor
DBG starting query execution qrid=GQJ1bYKYROk=
DBG local> run command sysctl -n machdep.cpu.features
DBG MPDgu3DNUgzD3BqWW2/FEclFrGsOFRJPdA+VnHrgcR6gLf0oKsGGyo1Naurh+CvQ2crTsoVOIN4HJVe3kcM9Gw== finished
DBG graph has received all datapoints
DBG finished query execution qrid=GQJ1bYKYROk=
cannot determine hypervisor
os.hypervisor: null
```

Reply:
Good call!